### ISSUE TITLE
LoadError on "using" statement in Julia 0.6

### ISSUE THREAD

**Issue by gondsm** [JuliaPOMDP/MCVI.jl#10]
Hello! I am trying to get the MCVI solver to work. So far I haven't had much success. As soon as I load the package, I get:

```
julia> using POMDPs

julia> using MCVI

WARNING: deprecated syntax "typealias Reward Float64" at /home/vsantos/.julia/v0.6/MCVI/src/MCVI.jl:10.
Use "const Reward = Float64" instead.
ERROR: LoadError: LoadError: TypeError: Type{...} expression: expected UnionAll, got Type{POMDPs.Updater}
Stacktrace:
 [1] include_from_node1(::String) at ./loading.jl:569
 [2] include(::String) at ./sysimg.jl:14
 [3] include_from_node1(::String) at ./loading.jl:569
 [4] eval(::Module, ::Any) at ./boot.jl:235
 [5] _require(::Symbol) at ./loading.jl:483
 [6] require(::Symbol) at ./loading.jl:398
while loading /home/vsantos/.julia/v0.6/MCVI/src/policy.jl, in expression starting on line 521
while loading /home/vsantos/.julia/v0.6/MCVI/src/MCVI.jl, in expression starting on line 17
```

This is on the latest version, of course. Is this some sort of incompatibility with Julia 0.6?

<sup>Am I perhaps doing something wrong...?</sup>

**Comment by zsunberg** [comment]
Hi @gondsm, I think you must somehow have an old version of the code. What is the output of
```
julia> cd(Pkg.dir("MCVI")); run(`git pull`)
```

**Comment by gondsm** [comment]
Hello @zsunberg, thanks for the reply!

Apparently when I said

> This is on the latest version, of course.

I was a bit... premature. I thought I had updated everything recently, but "recently" meant some time ago, possibly around the time Julia 0.6 first came out, which kind of feels like yesterday. Time flies! Turns out updating the repo (using your command, in fact!) instantly fixed the problem. 

Sorry for the unnecessary issue, closing it now, and many thanks once again!
